cockroachcloud/cloud-org-sso.md

```diff
@@ -0,0 +1,126 @@
+
+Refer to [Require SSO](configure-cloud-org-sso.html#require-sso).
+
+## Frequently Asked Questions (FAQ)
+
+#### Will it work if I try to sign in with SSO without explicitly setting it up for my account, if I already use an email from an SSO Provider such as Gmail?
+
+Yes, as long as the email address associated with your SSO provider is exactly the same as the one associated with your existing {{ site.data.products.db }} account. After successfully signing in, you will be prompted to approve the updated authentication method for your account.
+
+To view your current authentication method, visit **My Account** in the [{{ site.data.products.db }} Console](https://cockroachlabs.cloud).
+
+#### Once I change my active login method to a new SSO provider, can I still sign in using my email/password combination or my GitHub identity?
+
+No. Only one authentication method can be active for each {{ site.data.products.db }} Console user. Visit **My Account** in the [{{ site.data.products.db }} Console](https://cockroachlabs.cloud) to view or update your active authentication method.
+
+#### Does this change how administrators invite users?
+
+The [workflow for inviting team members](console-access-management.html#invite-team-members-to-cockroachdb-cloud) to {{ site.data.products.db }} remains the same. However, if Cloud Organization SSO is enabled for your {{ site.data.products.db }} organization and autoprovisioning is enabled for the authentication method they use to sign in, then an account is created automatically after they successfully authenticate.
+
+#### As an admin, how do I deprovision a user's access to {{ site.data.products.db }} Console if they leave the relevant project?
+
+If Cloud Organization SSO is enabled, then deprovisioning a user at the level of the IdP also removes their access to the {{ site.data.products.db }} organization.
+
+To remove a user's access to {{ site.data.products.db }} without deprovisioning the user from the IdP (such as when a user changes teams but does not leave the organization entirely), you can [remove their {{ site.data.products.db }} user identity from your {{ site.data.products.db}} organization](console-access-management.html#delete-a-team-member).
+
+#### Can admins require a particular authentication method for their {{ site.data.products.db }} organization?
+
+Yes, when Cloud Organization SSO is enabled for your {{ site.data.products.db }} organization, only the authentication methods you have enabled are displayed to your users.
+
+#### Which authentication flows are supported with Enterprise Authentication? Is it possible to enable the identity provider initiated flow?
```

Comment:
Is Enterprise Authentication a product feature? Why is it capitalized?

Reply:
We should change that to Cloud Organization SSO as that's the new name.